open_event/helpers/data.py

```diff
@@ -190,6 +194,9 @@ def create_sponsor(form, event_id):
         :param form: view data form
         :param event_id: Sponsor belongs to Event by event id
         """
+        file = request.files["logo"]
+    	filename = secure_filename(file.filename)
+	file.save(os.path.join(os.path.realpath('.') + '/static/Sponsor Logo/', filename))
```

Comment:
Let's not have files/directories with spaces in them. I suggest "sponsor_logos" but anything without a space works for me. Also, indentation